openslides/users/static/js/users/base.js

```diff
@@ -55,6 +55,10 @@ angular.module('OpenSlidesApp.users', [])
                 }
                 return _.intersection(perms, operator.perms).length > 0;
             },
+            // Returns true if the operator is a member of group.
+            isInGroup: function(group) {
+                return jQuery.inArray(group.id, operator.user.groups_id) > -1;
```

Comment:
Proposal: Use lodash instead of jQuery for consitency (`_.indexOf(operator.user.groups_id, group.id)`).